### PR TITLE
system-security-plan for ATO

### PR DESCRIPTION
**Pull request by jseppi**
Part of the ATO checklist is creating a `system-security-plan.yml` file.
This is a work-in-progress as we figure out how to properly fill this out.

ATO issue: https://github.com/18F/Infrastructure/issues/649

### CODE REVIEW

File: system-security-plan.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    org: General Services Administration
+    unit: 18F
+    email: devops@gsa.gov
+  system-security-officer:
+    name: Noah Kunin
+    title: 18F Infrastructure Director
+    org: General Services Administration
+    unit: 18F
+    email: devops@gsa.gov
+  technical-lead:
```
Comment (jseppi) [inline system-security-plan.yml]: @toolness - want to be the technical lead?

File: system-security-plan.yml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+information-types:
+- <Dxx + name>
+
+# https://pages.18f.gov/before-you-ship/ato/levels/
+confidentiality: low
+integrity: low
+availability: low
+security-baseline: low
+
+system-type: <major|minor|general support>
```
Comment (jseppi) [inline system-security-plan.yml]: Based on the example's linked at https://pages.18f.gov/before-you-ship/ato/ssp/#template, I guess this should be `minor`.

File: system-security-plan.yml (hunk 1 above)
Comment (toolness) [inline system-security-plan.yml]: Sure!

File: system-security-plan.yml [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+- D26 Civilian Operations
+
+# https://pages.18f.gov/before-you-ship/ato/levels/
+confidentiality: low
+integrity: low
+availability: low
+security-baseline: low
+
+system-type: minor
+level-of-identity-assurance: <0 if no authentication, or a link to SSP of the forthcoming identity system>
```
Comment (jseppi) [inline system-security-plan.yml]: I asked about this in the #compliance channel. Will update if I get a response.

File: system-security-plan.yml [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+environments:
+- Cloud Foundry
+- Amazon Web Services East
+user-types:
+  developer:
+    functions:
+    - deployment
+    - engineering
+controls:
+- <links to your security documentation>
```
Comment (jseppi) [inline system-security-plan.yml]: No clue what to put here

**Comment by jseppi** [comment]
I think this is ready to be merged now. We can make any modifications/additions in separate PRs.

**Comment by jseppi** [comment]
Merging because it has no code impact.

**Pull request merged**